example/next-storage/components/Avatar.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
+import { supabase } from '../lib/api'
+import styles from '../styles/Avatar.module.css'
+
+export type UploadButtonProps = {
+  avatar?: string | null
+}
+
+export default function Avatar(props: UploadButtonProps) {
+  const [avatarUrl, setAvatarUrl] = useState<string | null>(null)
+  const { avatar } = props
+
+  useEffect(() => {
+    async function downloadImage(path: string) {
```

Comment:
Error here (FF):

![image](https://user-images.githubusercontent.com/10214025/111600417-5ca16c00-880c-11eb-9171-edfb95d14591.png)

Debugging now

Reply:
wrapped it in a try/catch - data was null, error was `error The requested bucket was not found`

Reply:
Oh yeah, need to setup the bucket on supabase project first.

Maybe we should extract the bucket name to constants.ts

Reply:
I'll add it to the schema in the readme, but I'm still trying to make this work with a project on staging. Do you have any Policies enabled for your Buckets/objects?

Reply:
I have to disable RLS on `objects` table. 
https://supabase.slack.com/archives/CS7RWBQ8K/p1616040418006300